coderd/audit/audit.go

```diff
@@ -68,3 +72,76 @@ func (a *MockAuditor) Export(_ context.Context, alog database.AuditLog) error {
 func (*MockAuditor) diff(any, any) Map {
 	return Map{}
 }
+
+// Contains returns true if, for each non-zero-valued field in expected,
+// there exists a corresponding audit log in the mock auditor that matches
+// the expected values. Returns false otherwise.
+func (a *MockAuditor) Contains(t testing.TB, expected database.AuditLog) bool {
```

Comment:
Might be useful to be able to require multiple log entries and thus, also that the order is correct? Just a thought, feel free to disregard if it's premature.

Reply:
We could assert relative ordering perhaps, but asserting absolute ordering would likely run into the same kinds of issues we saw in https://github.com/coder/coder/issues/10396.